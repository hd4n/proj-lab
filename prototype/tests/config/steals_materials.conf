
@fields
em e1
em e2

@items
am a1
am a2
nu n1

@neighbors
e1 e2

@virologists
v1 e1 null a1;a2;n1
v2 e2