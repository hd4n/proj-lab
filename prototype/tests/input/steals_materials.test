load steals_materials.conf
steal v2 v1 materials
getstats v1
getstats v2