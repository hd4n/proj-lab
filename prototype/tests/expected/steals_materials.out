Load: sikeres
steal: v2 target: v1, materials

stats: v1
	fi_e1
	di_null
	st_false
	res_0.0
	re_false
	ma_10
	ref_0
	neededAmino_-1
	neededNucleo_-1
	co_null
	ma_a1+a2+n1
	ag_null
	eq_null
	eqa_null
	ef_null

stats: v2
	fi_e2
	di_null
	st_false
	res_0.0
	re_false
	ma_10
	ref_0
	neededAmino_-1
	neededNucleo_-1
	co_null
	ma_null
	ag_null
	eq_null
	eqa_null
	ef_null
